### DISCUSSION TITLE
How to add some whitespace to the bottom of an image

### DISCUSSION THREAD

**Discussion by dwg1957** [PintaProject/Pinta#776] [Q&A] [answered]
I'm new to Pinta and just cannot work out how to add some whitespace to the bottom of an image.

I'm updating some documentation and need to replace some screen shots. On some of the images I need to add some whitespace at the bottom of the image so I can put a file reference in there, but just cannot seem to be able to work out how to do this.

Thanks for any guidance.

**Comment by cameronwhite** [comment]
I think you want to use `Image` -> `Resize Canvas` if you want to add some additional space without resizing the existing image contents

**Reply by dwg1957** [reply]
I did try that but I got this checkerboard pattern in the space that I do not know how to change rather than just whitespace .

**Reply by cameronwhite** [reply, marked as answer]
Right, the new pixels are likely transparent by default, which appears as the checkerboard pattern. To fill them in a different color like white, you can use the Paint Bucket tool

**Reply by dwg1957** [reply]
Thanks very much, it was that last step that I missed.
